common/server.cpp

```diff
@@ -133,6 +133,8 @@ AuthenticationResult Server::loginUser(Server_ProtocolHandler *session, QString
                 delete se;
 
                 users.value(name)->prepareDestroy();
+            } else {
+                qDebug() << "Active session and sessions table inconsistent, please validate session table information for user " << name;
```

Comment:
:shipit:

On 8/31/15, ctrlaltca <notifications@github.com> wrote:
>> @@ -133,6 +133,8 @@ AuthenticationResult
>> Server::loginUser(Server_ProtocolHandler *session, QString
>>                  delete se;
>>
>>                  users.value(name)->prepareDestroy();
>> +            } else {
>> +                qDebug() << "Active session and sessions table
>> inconsistent, please validate session table information for user " <<
>> name;
>
> This will end up in server.log, where it already gets prepended with a
> timestamp. :+1:
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Cockatrice/Cockatrice/pull/1448/files#r38350056